src/Helpers/SpecGenerator.js

```diff
@@ -1,9 +1,7 @@
 import specStore from '../components/Stores/SpecGenerator'
 
 function parseSpecService (service, isMain = false) {
-  let parsedService = {
-    command: formatCommand(service),
-  }
+  let parsedService = {}
```

Comment:
Wouldn't this remove `command` for the services too?

Reply:
Thats correct and should not be the case. Fixed 👍